phalcon/session/adapter.zep

```diff
@@ -145,23 +134,31 @@ abstract class Adapter implements AdapterInterface
 	}
 
 	/**
-	 * Sets a session variable in an application context
+	 * Returns active session id
 	 *
 	 *<code>
-	 * $session->set("auth", "yes");
+	 * echo $session->getId();
 	 *</code>
 	 */
-	public function set(string index, var value)
+	public function getId() -> string
 	{
-		var uniqueId;
+		return session_id();
+	}
 
-		let uniqueId = this->_uniqueId;
-		if !empty uniqueId {
-			let _SESSION[uniqueId . "#" . index] = value;
-			return;
-		}
+	/**
+	 * Get session name
+	 */
+	public function getName() -> string
```

Comment:
Can we use (as usual) tabs instead spaces?

Reply:
I think it is fixed. I cannot tell from this editor.